### PR TITLE
Adjust FretLink job offer

### PR DESCRIPTION
**Pull request by fmenou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Some of these companies support remote hires. Where that information is availabl
 [Digital Asset](https://digitalasset.com) | Switzerland, Australia, Hungary, USA | [Yes](https://digitalasset.com/careers.html) | Finance | [Reddit](https://www.reddit.com/r/haskell/comments/6p2x0p/list_of_companies_that_use_haskell/dknymrm/)
 [Facebook](https://www.facebook.com) | London, UK |  [Yes](https://www.facebook.com/careers/jobs/a0I1200000LT8aA) | Advertising | [Blog](https://code.facebook.com/posts/745068642270222/fighting-spam-with-haskell/)
 [Factis Research](http://www.factisresearch.com) | Freiburg, Germany | No | Medicare | [Github](https://github.com/factisresearch?language=haskell)
-[Fretlink](https://www.fretlink.com) | Paris, France | Yes | Logistics/Transportation | [Reddit](https://www.reddit.com/r/haskell/comments/6p2x0p/list_of_companies_that_use_haskell/dkpp1gg/)
+[FretLink](https://www.fretlink.com/jobs) | Paris, France | To some extend | Logistics/Transportation | [Reddit](https://www.reddit.com/r/haskell/comments/6p2x0p/list_of_companies_that_use_haskell/dkpp1gg/)
```
Comment (erkmos) [inline README.md]: Hiring to some `extent`? What does that mean? and prefer having the jobs link in the `Hiring` column

**Comment by erkmos** [comment]
Thank you for the update, but I had some comments and questions

File: README.md (hunk 1 above)
Comment (fmenou) [reply README.md]: Here you go :)

**Comment by fmenou** [comment]
Updated

**Pull request merged**